src/test/scala/projectEuler/LargestPrimeFactorCalculatorSpec.scala

```diff
@@ -0,0 +1,42 @@
+package projectEuler
+
+import org.scalatest.{Matchers, WordSpec}
+
+/**
+ * Created by sonia on 12/10/2015.
```

Comment:
We prefer not to use non-code related comments in the code - this information can be retrieved from git